### PR TITLE
[CALCITE-4724] clickhouse supportsAliasedValues should be false

### PR DESCRIPTION
**Pull request by Enzo-Liu**

### CODE REVIEW

**Review by zabetak — CHANGES_REQUESTED** [review]
Please add tests for this change in `RelToSqlConverterTest`.

**Comment by Enzo-Liu** [comment]
Updated with testcase for clickhouse `select 1` query.

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 1]
```diff
@@ -5986,6 +5986,18 @@ private void checkLiteral2(String expression, String expected) {
         .ok(expected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4724">[CALCITE-4724]
+   * As for ClickHouse do not support values in from clause.
+   */
```
Comment (NobiGo) [inline core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:5992]: 1. the test descript should be the same as the CALCITE-4724.
2. the java doc lack of end label about "</a>"

Comment (Enzo-Liu) [reply core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:6003]: Updated . 
And add two more case in `testValues()` and `testValuesEmpty()`, please review again. tks

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 2]
```diff
@@ -5986,6 +5997,18 @@ private void checkLiteral2(String expression, String expected) {
...
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4724">[CALCITE-4724]
+   * ClickHouseSqlDialect `supportsAliasedValues` should return false. </a>
+   */
+  @Test void testAliasedValueForClickHouse() {
+    final String query = "select 1";
+    final String expected = "SELECT 1";
+    sql(query)
+        .withClickHouse()
+        .ok(expected);
```
Comment (NobiGo) [inline core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:6009]: some tips:  sql(query).withClickHouse().ok(expected);  may be better.

**Review by NobiGo — APPROVED** [review]
LGTM

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java (hunk 2 above)
Comment (Enzo-Liu) [reply core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java]: adjusted.

**Pull request closed without merging**